pkg/apis/validation/validation.go

```diff
@@ -23,7 +23,7 @@ import (
 	"time"
 
 	cmapiutil "github.com/jetstack/cert-manager/pkg/api/util"
-	cmapi "github.com/jetstack/cert-manager/pkg/apis/certmanager/v1alpha2"
```

Comment:
Have to update this one import here due to us using some KeyUsages functions, but this is a no-op change as the referenced consts are identical between the two.